file-formats/cota/examples/sap_cota_testobject.cota.json

```diff
@@ -0,0 +1,12 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Object For Unit Tests",
+    "originalLanguage": "en"
+  },
+  "configuration": {
+    "communicationTargetClass": "SAP_COTA_TESTOBJECT",
+    "multitenancyMode": "crossClient"
+  },
+  "rfcSettings": {}
```

Comment:
@wurzka can you double check here? 'rfcSettings' is of type enum with a default.
I assume that the object has the default `rfcSettings` and hence, the field shall be omitted during serialization.

Reply:
Solved :)